examples/07_advanced/plot_beta_series.py

```diff
@@ -0,0 +1,382 @@
+========================================================================
+This example shows how to run beta series :term:`GLM` models, which are a
+common modeling approach for a variety of analyses of task-based fMRI
+data with an event-related task design, including
+:term:`functional connectivity`, :term:`decoding <Decoding>`, and
+representational similarity analysis.
+
+Beta series models fit trial-wise conditions, which allow users to create
+"time series" of these trial-wise maps, which can be substituted for the
+typical time series used in resting-state functional connectivity analyses.
+Generally, these models are most useful for event-related task designs,
+while other modeling approaches, such as psychophysiological interactions
+(PPIs), tend to perform better in block designs, depending on the type of
+analysis.
+See :cite:t:`cisler2014comparison` for more information about this,
+in the context of functional connectivity analyses.
+
+Two of the most well-known beta series modeling methods are
+Least Squares- All (LSA) :cite:p:`rissman2004measuring` and
+Least Squares- Separate (LSS)
+:cite:p:`mumford2012deconvolving,turner2012spatiotemporal`.
+In LSA, a single :term:`GLM` is run, in which each trial of each condition of
+interest is separated out into its own condition within the design matrix.
+In LSS, each trial of each condition of interest has its own :term:`GLM`,
+in which the targeted trial receives its own column within the design matrix,
+but everything else remains the same as the standard model.
+Trials are then looped across, and many GLMs are fitted,
+with the parameter estimate map extracted from each GLM to build the LSS beta
+series.
+
+.. topic:: Choosing the right model for your analysis
+
+    We have chosen not to reproduce analyses systematically comparing beta
+    series modeling approaches in nilearn's documentation;
+    however, we do incorporate recommendations from the literature.
+    Rather than taking these recommendations at face value, please refer back
+    to the original publications and any potential updates to the literature,
+    when possible.
+
+    First, as mentioned above, according to :cite:t:`cisler2014comparison`,
+    beta series models are most appropriate for event-related task designs.
+    For block designs, a PPI model is better suited- at least for
+    functional connectivity analyses.
+
+    According to :cite:t:`abdulrahman2016effect`,
+    the decision between LSA and LSS should be based on three factors:
+    inter-trial variability, scan noise, and stimulus onset timing.
+    While :cite:t:`mumford2012deconvolving` proposes LSS as a tool primarily
+    for fast event-related designs (i.e., ones with short inter-trial
+    intervals), :cite:t:`abdulrahman2016effect` finds, in simulations,
+    that LSA performs better than LSS when trial variability is greater
+    than scan noise, even in fast designs.
+"""
+# sphinx_gallery_thumbnail_number = -2
```

Comment:
I think this doesn't render as expected, right?

Reply:
It looks like it's using the second-to-last figure as the thumbnail, which is what I was aiming for.

Reply:
I meant that it shows up in the example, but I realize that this is the only example in Nilearn where we'd use this.
I think it is possible to avoid having these configuration comments showing up in the rendered example. It _should_ be as simple as [adding a line in the config](https://sphinx-gallery.github.io/stable/configuration.html#removing-config-comments).

Reply:
Ohhh... I didn't even realize that was an option! That's awesome. I'll try it out.